### ISSUE TITLE
Incompatability With Vistas

### ISSUE THREAD

**Issue by LudoCrypt** [sf-inc/music_control#1]
The mod does not work when [Vistas](https://www.curseforge.com/minecraft/mc-mods/vistas) is installed. Other features in this mod also exist in some of my other mods [Music Duration Reducer](https://www.curseforge.com/minecraft/mc-mods/music-duration-reducer) and perhaps [Dynamic Music](https://www.curseforge.com/minecraft/mc-mods/dynamic-music) too. 

Edit: to be more clear, when Vistas uses custom menu music, the custom menu music does not play.
Edit 2; This also does not work when custom dimensions use custom music. There are more than the normal music types, other mods can add more, you gotta sort that out.

**Comment by charlyb01** [comment]
My mod wasn't meant to play other music but Minecraft. I'll see what I can do to play other musics but this won't change the fact that if you want to play a specific one for the menu it won't play as it will be another category (any way that's not the purpose).

For your mod Music Duration Reducer: yes my mod allows this also even though that is not the main goal of the mod. Yours may be enough for some people, it won't affect your mod as mine has another purpose.

For Dynamic Music: this is completely different. Your mod basically adds new musics and changes music speed/pitch. Not related to my mod.

To sum up, I don't plan to do anything but including categories from other mod. For everything else, those are not bugs but just the normal behavior of the mod. I hope everything is clear now ;)

**Comment by charlyb01** [comment]
Do you have any mod adding musics suggestions? I can't find any...

**Comment by LudoCrypt** [comment]
if you're asking for mods that add custom music, Vistas (along with a proper resource pack) works, My mod The Backrooms also adds more music in its dimensions. I believe the Lint2 mod also has custom music. BetterEnd has its own music as well. Let's see… Cinderscapes has its own biome music (composed by _yours truly_ (and a few others)). If that is not what you're asking then I do not understand the question.
